class_book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  # Method to add a rental to the book's rentals
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] Great job working so far for adding the rental but I think it would be great if you add `person, date` as parameters in add rental method here, please don't forget to pass the book itself when creating the new rental achieving the 'has-many' association` between the book and the person so kindly pass them then you can create the new rental 💯✅⭐🙏🏻

you can do something like this 


```ruby
def add_rental(person, date)
  Rental.new(date, self, person)
  end
```